### PR TITLE
DWC Business Builder

### PR DESCRIPTION
**Pull request by cheg7**

### CODE REVIEW

**Comment by MichaelCzcz** [comment]
@cheg7 There are no files in your PR. What did you change?

**Comment by cheg7** [comment]
> @cheg7 There are no files in your PR. What did you change?

Hi @MichaelCzcz , I had added metadata to the tutorials as suggested by Daniel. It seems like we need actual changes in the file to be able to compare the changes. I now have modified something in each file and have 7 changes. Request you to merge it to production.

Thank you!

File: tutorials/data-warehouse-cloud-businessbuilder2-dataset/data-warehouse-cloud-businessbuilder2-dataset.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ primary_tag: products>sap-data-warehouse-cloud
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/LNT7ciCJnpA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-In this tutorial, we will use an example dataset that contains information about sales. Please keep in mind that any data that will be used in the Business Builder needs to first be connected to or imported into SAP Data Warehouse Cloud before you can start creating business objects with the Business Builder.
+In this tutorial, we will use an example dataset that contains information about sales. Please keep in mind that any data that will be used in the Business Builder needs to first be connected to or imported into SAP Data Warehouse Cloud before you can start creating business objects with Business Builder.
```
Comment (MichaelCzcz) [inline tutorials/data-warehouse-cloud-businessbuilder2-dataset/data-warehouse-cloud-businessbuilder2-dataset.md:29]: Please don't use the "royal we", say "you will use" instead.

File: tutorials/data-warehouse-cloud-businessbuilder3-dimension/data-warehouse-cloud-businessbuilder3-dimension.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ primary_tag: products>sap-data-warehouse-cloud
 - How to preview your data and adjust the perspective
 
 
-In this tutorial, we will use an example dataset that contains information about sales. Please keep in mind that any data that will be used in the Business Builder needs to first be connected to or imported into SAP Data Warehouse Cloud before you can start creating business objects with the Business Builder.
+In this tutorial, we will use an example dataset that contains information about sales. Please keep in mind that any data that will be used in the Business Builder needs to first be connected to or imported into SAP Data Warehouse Cloud before you can start creating business objects with Business Builder.
```
Comment (MichaelCzcz) [inline tutorials/data-warehouse-cloud-businessbuilder3-dimension/data-warehouse-cloud-businessbuilder3-dimension.md:25]: Please don't use the "royal we", say "you will use" instead.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/data-warehouse-cloud-businessbuilder3-dimension/data-warehouse-cloud-businessbuilder3-dimension.md (hunk 2 above)
Comment (cheg7) [reply tutorials/data-warehouse-cloud-businessbuilder3-dimension/data-warehouse-cloud-businessbuilder3-dimension.md]: Changes suggested have been made to the tutorials. Thank you!

**Pull request merged**